walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/OpenIdPresentationRequest.kt

```diff
@@ -35,16 +37,16 @@ internal class OpenIdPresentationRequest(
     }
 
     // Completes the presentation request and returns Result with success status if successful.
-    override suspend fun complete(): Result<Unit> {
-        return try {
-            val result = OpenIdResponder.sendPresentationResponse(request.rawRequest, requirement)
-            Result.success(result)
-        } catch (exception: WalletLibraryException) {
-            Result.failure(exception)
+    override suspend fun complete(): VerifiedIdResult<Unit> {
+        return getResult {
+            OpenIdResponder.sendPresentationResponse(request.rawRequest, requirement)
         }
     }
 
-    override suspend fun cancel(message: String?): Result<Unit> {
-        return Result.failure(PresentationRequestCancelIsNotSupported("Cancelling presentation request is not supported."))
+    override suspend fun cancel(message: String?): VerifiedIdResult<Unit> {
+        return UserCanceledException(
```

Comment:
nit: Should this just be wrapped in a getResult { } function you have in other external functions?

Reply:
done